Settings screen update to 0.14

el00ruobuob opened the pull request:
This is the update of the settings tab.  
It includes the layout view explained (for `payment ID` and `lock wallet`).

The seed & keys view height is so huge i had to add as second screenshot.

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 ![settings-wallet](media/black_settings-wallet.png)
 
+(1) **Close this wallet:** Click here to safely close the current wallet  
+(2) **Create a view-only wallet:** Click here to generate a view-only (or audit) wallet, which is capable to see existing outgoing transactions and all incoming transactions. See this [user-guide](https://getmonero.org/resources/user-guides/view_only.html) for more informations.  
```
>which is capable to see existing outgoing transactions and all incoming transactions

I'd remove the existing outgoing transactions part, as that only applies after key images have been imported. Thus, change to:

>Click here to generate a view-only (or audit) wallet, which is capable of seeing all incoming transactions to the wallet. 

Perhaps we could add something later about importing key images. 

"informations" <= Change to information.

el00ruobuob replied:
> I'd remove the existing outgoing transactions part, as that only applies after key images have been imported.

Aren't the key images from previous transactions already included when you create a view-only wallet from a regular wallet?

el00ruobuob replied:
> "informations" <= Change to information.

Ok.

dEBRUYNE-1 replied:
That kind of complicates matters, as it would see outgoing transactions if the transaction used outputs that were received prior to the creation date, but not if the outputs were received after the creation date. I'll leave it up to you what to choose :-P

el00ruobuob replied:
I'll pass it to @erciccione then. I like being precise, but i get your point.

erciccione replied:
Ok to keep it for me and then add the specification dEBRUYNE suggested. Maybe at the bottom of the article, to not make that section too heavy.

el00ruobuob replied:
You mean about "importing key images"? Then maybe in the chapter 8, after the notes about remote & bootstrap nodes? We could link here from chapter 3 (send Monero > Export key images) and from this chapter.

erciccione replied:
Yes, we could also add some more info about key images, or link to the moneropedia.

el00ruobuob replied:
Indeed, add a bunch of info about what key images are, why they are needed for an audit (or view only) wallet, links to moneropedia, and an `import key images` screenshot.  
I'll will just wait for this #119 #122 and this one to be merged to work on the chapter 8 update, and add links to them in the same PR.

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
+### Layout
+![settings_layout](media/black_settings-layout.png)
+
+The `layout` tab is where you control the GUI wallet behavior. Most important options are:
```
`layout` -> `Layout` 

"where you control the GUI wallet behavior" -> where you can adjust the settings of the GUI wallet

"Most important options are" -> The most important options are"

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
+
+The `layout` tab is where you control the GUI wallet behavior. Most important options are:
+
+(1) **Hide balance:** If you are running your wallet in public, you may want to hide it's balance. This could be useful for point of sales.  
```
"it's balance" -> its balance

"This could be useful for point of sales." -> This can be useful for point of sales systems.

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
+The `layout` tab is where you control the GUI wallet behavior. Most important options are:
+
+(1) **Hide balance:** If you are running your wallet in public, you may want to hide it's balance. This could be useful for point of sales.  
+(2) **Enable Payment ID:** Payment ID are disabled by default. Click here to enable it. This can be useful to make transfers to Exchanges, which requires a Payment ID.  
```
"Payment ID are disabled by default" -> "Long payment IDs are disabled by default" 

"This can be useful to make transfers to Exchanges, which requires a Payment ID. " -> This can be useful for transfers to exchanges that require a long payment ID such as Binance (at the time of writing).

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
+
+(1) **Hide balance:** If you are running your wallet in public, you may want to hide it's balance. This could be useful for point of sales.  
+(2) **Enable Payment ID:** Payment ID are disabled by default. Click here to enable it. This can be useful to make transfers to Exchanges, which requires a Payment ID.  
+(3) **Lock wallet:** Set this to automatically lock your wallet after N minutes of inactivity. This would avoid anyone to use your wallet while you are away from keyboard.
```
> This would avoid anyone to use your wallet while you are away from keyboard.

This ensures no one is able to use your wallet while you are away from keyboard.

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 
 (1) **Start/Stop Local Node:** Depending on its current state, either start or stop the local node.  
-(2) **Blockchain location:** Manually enter a non-default path to the blockchain.  
+(2) **Blockchain location:** Manually enter a non-default path to the blockchain, or click `change` to browse to a new path.  
```
"browse" -> choose

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 (3) **Startup flags:** When using a local node, this will be where you enter additional command line options.  
-(4) **Bootstrap Address:** Enter the hostname or IP address of the bootstrap remote node. See section [8.1 Bootstrap nodes](#8.1-bootstrap-nodes) for a brief explanation of what is a bootstrap node.  
-(5) **Bootstrap Port:** Enter the port of the bootstrap remote node.
+(4) **Bootstrap Address:** Enter the hostname or IP address of the bootstrap remote node. See section [Bootstrap nodes](#bootstrap-nodes) for a brief explanation of what is a bootstrap node.  
```
"of what is a bootstrap node. " <= of what a bootstrap node is

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 (3) **Daemon Username:**  enter a username in case authentication to the remote node is required.  
-(4) **Daemon Password:**  enter a password in case authentication to the remote node is required.
+(4) **Daemon Password:**  enter a password in case authentication to the remote node is required.  
+(5) **Trusted Daemon:** By default, data requests to remote nodes are obfuscated, and this process requires more ressources. If your remote node is under your control (i.e. a dedicated server of your own) you can mark it trusted so that data requests will not be obfuscated anymore (just like a local node).
```
"ressources" -> resources

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -39,10 +56,12 @@ Use a remote node, do not download the blockchain. Check the ['About remote node
 (5) **Wallet log path:** Where the logs for this wallet will be saved.  
-(6) **Copy to clipboard:** Copy all these informations to the clipboard.  
+(6) **Wallet mode:** Display the wallet mode chosen during the [wizard](#choose-wallet-mode).  
+(7) **Copy to clipboard:** Copy all these informations to the clipboard.  
```
"Copy all these informations to the clipboard" -> "Copy all information to clipboard"

el00ruobuob replied:
ok

dEBRUYNE-1 commented on en/ch06.md:
```diff
@@ -39,10 +56,12 @@ Use a remote node, do not download the blockchain. Check the ['About remote node
 (4) **Wallet creation height:** Change the block height that a wallet rescan will go back to.  
 (5) **Wallet log path:** Where the logs for this wallet will be saved.  
-(6) **Copy to clipboard:** Copy all these informations to the clipboard.  
+(6) **Wallet mode:** Display the wallet mode chosen during the [wizard](#choose-wallet-mode).  
```
"during" -> in

el00ruobuob replied:
ok

erciccione commented:
@el00ruobuob Please rebase :)

el00ruobuob commented:
I have no idea what i did here @erciccione 
It seems i didn't built it above #119 for some reason...
Well i rebased and moved the changed to ch08.md instead of ch06.md, so it should be ok now

erciccione commented on en/ch08.md:
```diff
@@ -39,10 +56,12 @@ Use a remote node, do not download the blockchain. Check the ['About remote node
 
 ## Seed and keys
 ![seed-keys](media/black_seed-keys.png)
+![seed-keys](media/black_seed-keys_2.png)
```
Are the two screenshots one after the other on purpose? I'm afraid that could make the format a bit messed up

el00ruobuob replied:
I'll give them some space.

el00ruobuob replied:
Does it look better?

erciccione replied:
That's fine, we can also fix the formatting later on
